dydx-testnet-4/gentx/gentx-LavenderFive.json

```diff
@@ -0,0 +1,62 @@
+{
+  "body": {
+    "messages": [
+      {
+        "@type": "/cosmos.staking.v1beta1.MsgCreateValidator",
+        "description": {
+          "moniker": " Lavender.Five Nodes :bee:",
+          "identity": "F87ADDB700C0CC94",
+          "website": "https://www.lavenderfive.com/",
+          "security_contact": "hello@lavenderfive.com",
+          "details": "100% soft slash protected. We strive to make the cosmos a more holistically wholesome place. Come say Hi! https://linktr.ee/lavenderfive"
+        },
+        "commission": {
+          "rate": "0.050000000000000000",
+          "max_rate": "0.100000000000000000",
+          "max_change_rate": "0.050000000000000000"
+        },
+        "min_self_delegation": "1",
+        "delegator_address": "dydx14qekdkj2nmmwea4ufg9n002a3pud23y8vmw038",
+        "validator_address": "dydxvaloper14qekdkj2nmmwea4ufg9n002a3pud23y8f9ce6k",
+        "pubkey": {
+          "@type": "/cosmos.crypto.ed25519.PubKey",
+          "key": "BvO4r6Hx08iciu63Y5YV+Pbk6x7oxT47GCJaUODP54s="
+        },
+        "value": {
+          "denom": "dv4tnt",
+          "amount": "50000000000"
```

Comment:
@reversesigh the [Prepare for Genesis](https://v4-teacher.vercel.app/validators/preparing_for_genesis) guide is a little different this time, the test token is `adv4tnt` and the self-delegation amount is `500000000000000000000000adv4tnt`

Reply:
@reversesigh kind reminder